docs/administration-guide/operation-score.md

```diff
@@ -0,0 +1,178 @@
+
+## Using Operation Scores
+
+A `ScoreOperationChain` operation determines a "score" for an OperationChain.
+
+Below is a simple example for constructing a ScoreOperationChain, using this directed graph:
+
+``` mermaid
+graph TD
+  1(1, count=3) -- count=3 --> 2
+  1 -- count=1 --> 4
+  2(2, count=1) -- count=2 --> 3
+  2 -- count=1 --> 4(4, count=1)
+  2 -- count=1 --> 5(5, count=3)
+  3(3, count=2) -- count=4 --> 4
+```
+
+!!! example "Example ScoreOperationChain"
+
+    === "Java"
+        ``` java
+        final ScoreOperationChain scoreOpChain = new ScoreOperationChain.Builder()
+            .operationChain(new OperationChain.Builder()
+                    .first(new GetElements())
+                    .then(new NamedOperation.Builder<Element, Iterable<? extends Element>>()
+                            .name("namedOp")
+                            .build())
+                    .then(new Limit<>(3))
+                    .build())
+            .build();
+        ```
+
+    === "JSON"
+        ``` json
+        {
+            "class" : "ScoreOperationChain",
+            "operationChain" : {
+                "class" : "OperationChain",
+                "operations" : [ {
+                    "class" : "GetElements"
+                }, {
+                    "class" : "NamedOperation",
+                    "operationName" : "namedOp"
+                }, {
+                    "class" : "Limit",
+                    "resultLimit" : 3,
+                    "truncate" : true
+                } ]
+            }
+        }
+        ```
+
+    === "Python"
+        ``` python
+        g.ScoreOperationChain( 
+            operation_chain=g.OperationChain( 
+                operations=[ 
+                    g.GetElements(), 
+                    g.NamedOperation( 
+                        operation_name="namedOp" 
+                    ), 
+                    g.Limit( 
+                        result_limit=3, 
+                        truncate=True 
+                    ) 
+                ] 
+            ) 
+        )
+        ```
+
+The key variables when using ScoreOperationChain are:
+
+- `opScores` - required map of operation scores. These are the operation score values.
+- `authScores` - required map of operation authorisation scores. These are the maximum scores allowed for a user with a given role. 
+- `scoreResolver` - some operations may require a custom way of calculating an associated score, therefore an implementation of the [`ScoreResolver`](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/store/operation/resolver/ScoreResolver.html) interface may be required. 
+These map the operation class to its respective score resolver.
+There is a `DefaultScoreResolver` to which the custom implementation should delegate, in a manner specific to the new Operation.
```

Comment:
It would be good to state which operations these are for "some operations may require a custom way". I think there's only a few of these - this could go in a further subsection.